lib/strip_js.ex

```diff
@@ -179,61 +183,59 @@ defmodule StripJs do
   possible for innocent CSS containing either of the strings `javascript:`
   or `expression(` to be mangled.
   """
-  @spec clean_css(String.t, opts) :: String.t
+  @spec clean_css(String.t(), opts) :: String.t()
   def clean_css(css, _opts \\ []) when is_binary(css) do
     css
     |> String.replace(~r/javascript \s* :/xi, "removed_by_strip_js:")
     |> String.replace(~r/expression \s* \(/xi, "removed_by_strip_js(")
   end
 
-
   ## Removes JS vectors from the given HTML attributes.
-  @spec clean_attrs([{String.t, String.t}]) :: [{String.t, String.t}]
+  @spec clean_attrs([{String.t(), String.t()}]) :: [{String.t(), String.t()}]
   defp clean_attrs(attrs) do
     attrs
     |> Enum.reduce([], &clean_attr/2)
-    |> Enum.reverse
+    |> Enum.reverse()
   end
 
   @attrs_with_urls ["href", "src", "background", "dynsrc", "lowsrc"]
 
-  @spec clean_attr({String.t, String.t}, [{String.t, String.t}]) :: [{String.t, String.t}]
+  @spec clean_attr({String.t(), String.t()}, [{String.t(), String.t()}]) :: [
+          {String.t(), String.t()}
+        ]
   defp clean_attr({attr, value}, acc) do
     attr = String.downcase(attr)
+
     cond do
-      (attr in @attrs_with_urls) && String.match?(value, ~r/^ \s* javascript \s* :/xi) ->
-        [{attr, "#"} | acc]  # retain the attribute so we emit valid HTML
+      attr in @attrs_with_urls &&
+          String.match?(value, ~r/^ \s* javascript \s* :/xi) ->
+        # retain the attribute so we emit valid HTML
+        [{attr, "#"} | acc]
+
       String.starts_with?(attr, "on") ->
-        acc  # remove on* handlers entirely
+        # remove on* handlers entirely
+        acc
+
       :else ->
-        [{attr, value |> escape_quotes |> html_escape} | acc]
+        [{attr, value} | acc]
```

Comment:
we don't have to do this anymore?

Reply:
nope, my patch means floki does this by default now (for data attributes)

Reply:
by "this" I mean escaping quotes to `&quot;` along with less than, greater than, and apostrophes